sympy/algebras/tests/test_quaternion.py

```diff
@@ -172,3 +172,15 @@ def test_quaternion_multiplication():
 
     assert z * q == z_quat * q
     assert q * z == q * z_quat
+
+
+def test__rtruediv__iss16318():
+    q1 = Quaternion(1, 2, 3, 4)
+    q0 = Quaternion(0, 0, 0, 0)
```

Comment:
Please use the naming of function as `def test_issue_16318`, to maintain the uniformity across all modules. The same applies for the next test function name.

Reply:
I made the changes. Since both of this tests are related to the same issue, i put them under one function named test_issue_16318. Is it ok now?
Thank you for your help!